packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -310,7 +310,7 @@ const PostsPage = ({post, eagerPostComments, refetch, classes}: {
   const currentUser = useCurrentUser();
   const { openDialog } = useDialog();
   const { recordPostView } = useRecordPostView(post);
-  const [highlightDate,setHighlightDate] = useState<Date|undefined>(post?.lastVisitedAt && new Date(post.lastVisitedAt));
+  const [highlightDate,setHighlightDate] = useState<Date|undefined|null>(post && post.lastVisitedAt && new Date(post.lastVisitedAt));
```

Comment:
Huh, did it not like `post?.lastVisitedAt && new Date(post.lastVisitedAt)`?  Am surprised.

Reply:
It didn't seem to, but now it seems to. Perhaps it was my TS integration being slow